resources/compass/templates/tenant-fetcher-job.yaml

```diff
@@ -9,8 +9,14 @@ spec:
   concurrencyPolicy: Forbid
   startingDeadlineSeconds: 60 # Fix for https://github.com/kyma-incubator/compass/issues/1155
   jobTemplate:
+    metadata:
+      labels:
+        cronjob: tenant-fetcher
     spec:
       template:
+        metadata:
+          labels:
+            cronjob: tenant-fetcher
```

Comment:
I didn't use the labels after all (I planned to do so), but anyway I think it may be handy to have them here - to quickly filter tenant-fetcher jobs.